ui/app/pages/settings/contact-list-tab/index.scss

```diff
@@ -21,7 +21,7 @@
     border-bottom: 1px solid #dedede;
 
     &:hover {
-      border: 1px solid #037DD6;
```

Comment:
A lot of these changes seem unrelated to the primary purpose of this PR. Is this the work of some auto-formatter or something? :thinking: 

This particular change (lower-casing hex colours) is in-line with our current stylelint config (which is broken but will be fixed shortly), but some of the other changes (e.g. adding a leading zero to decimal numbers) are in conflict with it.

Reply:
stylelint has been re-renabled for all auto-fixable rules in #8989, which was just merged. So most of these changes should vanish now if you rebase on `develop`,